src/main/webapp/index.html

```diff
@@ -85,7 +85,9 @@ <h1>TheGlobetrotter</h1>
                   placeholder="Marker name"
                 />
 
-                <div class="time-text">Time in place</div>
+                <div class="time-text">
```

Comment:
Are you sure it is necessary? Such a long line knocks down centering of objects 🤔

Reply:
Well, I just thought that it is a little bit clearer like that

Reply:
maybe but it just... really long...

Reply:
Is it better if I change it to "Time to spend here:"?

Reply:
What about "Spend time on the spot"? 💦  it is more clear than previous one and a bit shorter

Reply:
oh damn, sorry didn't notice 💦 💦 💦 💦 

>Is it better if I change it to "Time to spend here:"?
Yes!